### DISCUSSION TITLE
biomes are shown wrong

### DISCUSSION THREAD

**Discussion by Blook888** [Ceddini/CoreKeeperMapTool#31] [Q&A]
hi everyone.

currently i am on the journey to find the titan temple. while exploring and updating the map i realized that somehow the biome background of this map feature is suddenly completely wrong/different than before.
![corekeeper_map](https://github.com/Ceddini/CoreKeeperMapTool/assets/136812745/55cec7ba-24bb-462f-a2f2-7adfdbda3979)

in this picture you can see at the bottom that some part of the desert biome is shown being in the wilderness biome. i am slao to 99% sure that the lava biome in the north-west part of the discovered desert was at the edge and not in the middle of the esert biome.

and no, i didnt changed the angle manually.

hope someone can help me.

**Comment by Ceddini** [comment]
It's based on different pixel counts between the biomes, so you'd just have to explore more in the desert and the other biomes for it to become more accurate.
So you probably were just lucky before. If it's like this, please just use the manual rotation :)

I wanna move the whole thing over to a different framework at some point and will probably rewrite some of the legacy stuff then, but for now i kinda lack the time
